test/cli/seed-make.spec.js

```diff
@@ -395,6 +395,7 @@ development: {
 
     it('Creates a new seed using --timestamp-filename-prefix CLI flag', async () => {
       const seedGlobPath = `${process.cwd()}/seeds/*_somename.js`;
+      console.log(seedGlobPath);
```

Comment:
is this necessary?

Reply:
nope, thought i got rid of that. apparently not. thanks for the catch, will fix now.